rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -256,13 +246,44 @@ def encode(self, bboxes_in, labels_in):
     def GetOneHotEncodedLabels(self, array):
         return b.getOneHotEncodedLabels(self._handle, array, self._numOfClasses)
 
+    def set_outputs(self, *output_list):
+        self._output_list_length = len(output_list)
+        b.setOutputImages(self._handle,len(output_list),output_list)
+
+    def __enter__(self):
+        Pipeline._current_pipeline = self
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback):
+        pass
+
+    def set_seed(self,seed=0):
+        return b.setSeed(seed)
+
+    def create_int_param(self,value):
```

Comment:
I think it is complaining because it doesn't need to be a member of class in this case. Does this need to be a classmember? Please follow the link below to specify the function decorator type
https://docs.quantifiedcode.com/python-anti-patterns/correctness/method_could_be_a_function.html

Reply:
Yes, this needs to class members since we use these functions in the augmentation app access via pipeline method